ssl-config/build.gradle

```diff
@@ -21,3 +29,45 @@ task generateCerts(type:Exec) {
 
 tasks.check.dependsOn(javadoc)
 tasks.test.dependsOn(generateCerts)
+
+dockerTestRunner {
+    // missing BouncyCastle but contains Sun libraries
+    dockerRunner name: 'openjdk-7', dockerFile: file('src/test/resources/openjdk-7.dockerfile')
+
+    // missing BouncyCastle and Sun libraries
+    dockerRunner name: 'ibmjdk-8', dockerFile: file('src/test/resources/ibmjdk-8.dockerfile')
+
+    // run only the PKCS1 tests and exclude BouncyCastle runtime dependency
+    testConfig = {
+        include('**/pkcs1/*Tests*')
```

Comment:
True. Thoughts on a better approach? Seems like making an entirely new "integration test" type project would be over-kill. Maybe it would be better to add logic to the build such that this will throw an exception if there are no files present (so that failure on rename won't be silent)?